labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -65,8 +83,7 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+    seq == seq.distinct && seq == seq.sorted
```

Comment:
Sorting a list takes O(n log n) time in the worst case and is at least linear (even if it’s already sorted you need to check); going through and checking if it is sorted is at most linear (and can be less if you can quit early).